libindy/src/commands/mod.rs

```diff
@@ -80,9 +96,14 @@ pub fn indy_set_runtime_config(config: IndyConfig) {
     }
 }
 
+fn get_cur_time() -> u64 {
+    let since_epoch = SystemTime::now().duration_since(UNIX_EPOCH).expect("Time has gone backwards");
+    since_epoch.as_secs()
```

Comment:
Seconds resolution is not enough for these kind of measurements, we need at least milliseconds.